### PR TITLE
Alvin

### PR DESCRIPTION
**Pull request by chen-alvin**

### CODE REVIEW

File: alvin_c/server.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+})
+mongoose.model('Meese', MeeseSchema); // We are setting this Schema in our Models as 'meese'
+var Meese = mongoose.model('Meese') // We are retrieving this Schema from our Models, named 'meese'
+mongoose.Promise = global.Promise; // because reasons
+
+// Routes //
+// display all of the meese
+app.get('/', function(req, res) {
+  Meese.find({}, function(err, meese){
+    if(!err){
```
Comment (brendan-codes) [inline alvin_c/server.js:28]: We recommend to write 'truthy' statements. This is nothing more than a good coding habit that a lot of developers subscribe to. If(err) is strangely prefered to if(!err).

File: alvin_c/server.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    }
+  })
+})
+// should show a form to edit an existing meese
+app.get('/meese/:id/edit', function(req, res){
+  console.log('edit path called'+req.params.id)
+  Meese.find({_id:req.params.id}, function(err, meese){
+    if(!err){
+      //console.log(meese);
+      res.render('form1',{meese:meese});
```
Comment (brendan-codes) [inline alvin_c/server.js:62]: Ideally, the view for a new meese would be called 'new.ejs'. Remember, your view names will be available on the client, so every user of your site will be able to read 'form1' through inspect element.

File: alvin_c/views/form1.ejs [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  </head>
+
+  <body>
+    <div class='container'>
+      <div class='col-sm-8 col-sm-offset-2'  style='padding:10px'>
+        <a href='/'><h2>Meese Dash</h2></a>
+        <h4>Edit a meese</h4>
+        <table class='table'>
+          <tr><th>Name</th><th>Age</th></tr>
+          <%for (x in meese){%>
```
Comment (brendan-codes) [inline alvin_c/views/form1.ejs:18]: Always declare variables with the var keyword, even in for loops, even on ejs.

**Review by brendan-codes — APPROVED** [review]
Overall, this is an excellent Mongoose Dash. I have a couple nit-picky comments in there that I recommend reviewing, but I'm going to approve this request. Great job, keep it up Alvin!

**Comment by chen-alvin** [comment]
Thank you! I didn't know about these stylistic choices but will revise my code and implement in future code.

**Pull request merged**